router/parse.js

```diff
@@ -1,12 +1,14 @@
-import Url from 'lite-url';
 import withTrailingSlash from './with-trailing-slash';
 
-const PROTOCOL_REGEX = /https?:\/\//;
+const PANELS_REGEX = /^https?:\/\/([a-zA-Z0-9\-\_\.]+)()(\/.*)/;
```

Comment:
Could you add a comment to explain what the regex's groups are there for? Something like the fact that they work as a movable splitting point? (needs better words than that :P) Thanks!